rx/core/observable/marbles.py

```diff
@@ -91,26 +91,30 @@ def action(scheduler, state=None):
     return Observable(subscribe)
 
 
-def from_marbles(string: str, timespan: RelativeTime = 0.1, lookup: Dict = None,
-                 error: Optional[Exception] = None, scheduler: Optional[Scheduler] = None) -> Observable:
+def from_marbles(string: str,
+                 timespan: RelativeTime = 0.1,
+                 lookup: Dict = None,
```

Comment:
Should the Dict have generic type vars (and same in the `parse` function)? I guess we can't do any better than `Dict[Union[int, float, str], Any]` though, so probably not worth it.

Reply:
Good catch! We can even use `Mapping` instead. I've also missed `Optional` hint for `lookup`.